.golangci.yml

```diff
@@ -20,6 +20,7 @@ linters:
     - gci # too strict
     - contextcheck # too restrictive
     - gomoddirectives # TODO remove after scany/tern v2 are released
+    - musttag # cache uses JSON for models (Account)
```

Comment:
Newer version of linter finds a problem with our JSON missing tag, will fix in a different PR.